### PR TITLE
Migrate rest of drink mutagens

### PR DESCRIPTION
**Pull request by Cyrano7**
Copy of https://github.com/CleverRaven/Cataclysm-DDA/pull/16600

### CODE REVIEW

File: data/json/items/comestibles/mutagen.json [hunk 1]
```diff
@@ -20,75 +20,158 @@
         "flags" : [ "MUTAGEN_STRONG" ]
     },
     {
-        "id" : "mutagen_beast",
+        "id" : "mutagen_flavor", "//" : "meta item, not spawned in game but used as a baseline for class of mutagens",
```
Comment (mugling) [inline data/json/items/comestibles/mutagen.json]: This is an `abstract` item then

File: data/json/items/comestibles/mutagen.json [hunk 2]
```diff
@@ -20,75 +20,158 @@
         "flags" : [ "MUTAGEN_STRONG" ]
     },
     {
-        "id" : "mutagen_beast",
+        "abstract" : "mutagen_flavor", "//" : "meta item, not spawned in game but used as a baseline for class of mutagens",
```
Comment (Cyrano7) [inline data/json/items/comestibles/mutagen.json]: @mugling Besides the abstract part is the description fine? or should I just delete it.

Comment (mugling) [inline data/json/items/comestibles/mutagen.json]: Drop the comment, all `abstract` items have the semantics it describes

**Comment by Coolthulhu** [comment]
JSON errors during load.

**Comment by Cyrano7** [comment]
@Coolthulhu was that the only error?

**Comment by Coolthulhu** [comment]
I think so

**Comment by mugling** [comment]
Yes, but @Coolthulhu I just hit the same JSON error in the @pisskop mod (I'm presuming you fixed this in `master`?)

**Pull request merged**